src/mlpack/core/data/confusion_matrix.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file confusion_matrix.hpp
+ * @author Jeffin Sam
+ *
+ * Function for confusion matrix
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_CONFUSION_MATRIX_HPP
+#define MLPACK_CORE_DATA_CONFUSION_MATRIX_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace data {
+/**
+ * A confusion matrix is a summary of prediction results on a classification
+ * problem.The number of correct and incorrect predictions are summarized
+ * with count values and broken down by each class.
+ *
+ * @param pred vector of predicted values.
```

Comment:
Do you mind to go with ``predictors`` and ``responses`` to be consistent with the rest of the codebase;  might be more intuitive if we use the same names as for the classifiers.

Reply:
changed the param names